include/api.php

```diff
@@ -42,647 +42,658 @@
 $API = array();
 $called_api = null;
 
-/// @TODO Fix intending
```

Comment:
Nice! 😄

Reply:
I knew you would love this!!

Reply:
You removed a TODO and a typo in one fell swoop! 😂